cpp/src/gandiva/filter.cc

```diff
@@ -78,6 +79,10 @@ std::string FilterCacheKey::ToString() const {
   return ss.str();
 }
 
+std::string FilterCacheKey::GetExpressionsAsString() const {
```

Comment:
nit: GetExpressionAsString?

Reply:
This function is to be used only in the base_cache_key so it also stores the expressions and compare them when comparing one base_cache_key against another.

Reply:
yes I understood that. I meant the name here should be getExpression instead of getExpressions

Reply:
There is no need for it anymore, as it were only used on the ExpressionCacheKey and now the ExpressionCacheKey is already built with the expressions from projector or filter, not the old ProjectorCacheKey or FilterCacheKey